templates/server.properties.j2

```diff
@@ -37,18 +38,17 @@ background.threads={{ kafka_background_threads }}
 
 ############################# Socket Server Settings #############################
 
-# The address the socket server listens on. It will get the value returned from
-# java.net.InetAddress.getCanonicalHostName() if not configured.
+# The address the socket server listens on. If not configured, the host name will be equal to the value of
```

Comment:
Updates to comments based on diff'ing the latest config from the Kafka 3.2.0 release.